README.md

```diff
@@ -142,7 +142,7 @@ Storage Options
 ---------------
 
 ### Local
-Sccache defaults to using local disk storage. You can set the `SCCACHE_DIR` environment variable to change the disk cache location. By default it will use a sensible location for the current platform: `~/.cache/sccache` on Linux, `%LOCALAPPDATA%\Mozilla\sccache` on Windows, and `~/Library/Caches/Mozilla.sccache` on MacOS.
+Sccache defaults to using local disk storage. You can set the `SCCACHE_DIR` environment variable to change the disk cache location. By default it will use a sensible location for the current platform: `~/.cache/sccache` on Linux, `%LOCALAPPDATA%\Mozilla\sccache` on Windows, and `~/Library/Caches/Mozilla.sccache` on MacOS. To limit the cache size set `SCCACHE_CACHE_SIZE`, for example `SCCACHE_CACHE_SIZE="1G"SCCACHE_CACHE_SIZE`
```

Comment:
I think it would be good to document the default here.

The "``SCCACHE_CACHE_SIZE="1G"SCCACHE_CACHE_SIZE`" bit at the end looks wrong and needs a period at the end of the sentence.